lib/push.js

```diff
@@ -422,7 +422,8 @@ module.exports = function (log, db, config) {
                 // set the push endpoint expired flag
                 // Warning: this method is called without any session tokens or auth validation.
                 device.pushEndpointExpired = true
-                return db.updateDevice(uid, null, device).catch(function (err) {
+                device.sessionTokenId = null
```

Comment:
I'm not sure I understand what's happening here - is this explicitly setting the `sessionTokenId` to `null`, or is it just trying to update the `pushEndpointExpired` field without overwriting the `sessionTokenId`?  From reading the surrounding code it *seems* like it should be doing the later, but passing an explicit `null` make me think it's doing the former.

Reply:
we can just delete this probably